.github/workflows/ansible-lint.yaml

```diff
@@ -0,0 +1,19 @@
+# https://github.com/openstack-k8s-operators/osp-director-dev-tools/blob/dae8a937bf56a3d50e4a1ffa60b47a485b5baa22/.github/workflows/ansible-lint.yaml#L2
+name: Ansible Lint
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+jobs:
+  test:
+    name: Ansible Lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Lint Ansible Playbooks
+        uses: ansible/ansible-lint-action@v6.0.2
```

Comment:
this needs update. apparently we should now use that instead https://github.com/marketplace/actions/run-ansible-lint